tests/test_api_push_validate.py

```diff
@@ -380,6 +380,8 @@ def test_10_enroll_push(self):
         # 1. set policies.
         set_policy("pol_passthru", scope=SCOPE.AUTH, action=ACTION.PASSTHRU)
 
+        set_policy("pol_tokenlabel", scope=SCOPE.ENROLL, action="{0!s}=Pushy".format(ACTION.TOKENLABEL))
```

Comment:
i don't see an assert to verify the modified label, am i missing something?

Reply:
I do not see a way to verify this besides debugging.